### ISSUE TITLE
annotate_raw_with_fastqs fails with KeyError

### ISSUE THREAD

**Issue by tillbaar** [nanoporetech/tombo#19]
Hi, I tried to use `tombo annotate_raw_with_fastqs` to exchange the sequence information in my FAST5 files. I have used MinKNOW's on-the-fly basecalling, so my FAST5 files do contain sequence information already, but I later redid the basecalling using Albacore. I would like to exchange this information, as I have used direct RNA sequencing and my MinKNOW-produced basecalls are composed of ACGU, but my template of ACGT. I plan on using `tombo resquiggle` next, using GraphMap as the aligner.

When I do this however:
```
tombo annotate_raw_with_fastqs \
--fast5-basedir /path/to/fast5dir \
--fastq-filenames /path/to/basecall.fastq \
--overwrite
```
I encounter the following error message:
```
Preparing reads and extracting read identifiers.
Annotating FAST5s with sequence from FASTQs.
********* WARNING ********
Some records from /path/to/basecall.fastq contain read identifiers not found in any FAST5 files.
Traceback (most recent call last):
  File "/home/user/apps/miniconda/envs/ve_tombo/bin/tombo", line 11, in <module>
    load_entry_point('ont-tombo==1.1.1', 'console_scripts', 'tombo')()
  File "/home/user/apps/miniconda/envs/ve_tombo/lib/python2.7/site-packages/tombo/__main__.py", line 158, in main
    tombo_helper.annotate_reads_with_fastq_main(args)
  File "/home/user/apps/miniconda/envs/ve_tombo/lib/python2.7/site-packages/tombo/tombo_helper.py", line 1584, in annotate_reads_with_fastq_main
    annotate_with_fastqs(args.fastq_filenames, fast5s_read_ids, fastq_slot)
  File "/home/user/apps/miniconda/envs/ve_tombo/lib/python2.7/site-packages/tombo/tombo_helper.py", line 487, in annotate_with_fastqs
    with h5py.File(fast5s_read_ids[read_id]) as fast5_data:
KeyError: '748f2b91-d711-456f-837d-d3780e7feefb'
```
I understand the `Some records from /path/to/basecall.fastq contain read identifiers not found in any FAST5 files.` part, as I am not using all the FAST5 files I have, but the rest I cannot decipher.

The result is, by the way, that the old sequence information is removed from my FAST5 files, but no new sequence information is added.

I hope that you can assist me with my problem.
Best, Till

**Comment by marcus1487** [comment]
This bug was actually just found and resolved just yesterday (#17). A fix has been pushed to the git repository for now (#18). The pip and conda packages will be updated at the next release.

So to fix this for now, you can install from the git installation instructions and your error should be resolved.

**Comment by tillbaar** [comment]
Thank you very much!
It works like a charm now.
